src/screens/InfoScreen/InfoScreen.tsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react';
+import DeviceInfo from 'react-native-device-info';
+import {Text, View} from 'react-native';
+import { useThemedStyles } from 'styles/commonStyles';
+import styles from 'src/screens/InfoScreen/InfoScreenStyles';
+
+
+export default function InfoScreen() {
+  const themedStyles = useThemedStyles();
+  const [appInfo, setAppInfo] = useState({ version: '', build: '' });
+
+  useEffect(() => {
```

Comment:
not need to have useEffect here and state as well
just use
const version = DeviceInfo.getVersion();
const build = DeviceInfo.getBuildNumber();
and remove state and useEffect